<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pathsForm</class>
 <widget class="QWidget" name="pathsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dictionaries</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="dictionaryComboBox"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="dictHorizontalLayout">
     <item>
      <widget class="QLineEdit" name="dictDirLineEdit">
       <property name="toolTip">
        <string/>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="dictDirToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Ydpd.qrc">
         <normaloff>:/res/document-open.png</normaloff>:/res/document-open.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="sampleHorizontalLayout">
     <item>
      <widget class="QLineEdit" name="sampleDirLineEdit">
       <property name="toolTip">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="sampleDirToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Ydpd.qrc">
         <normaloff>:/res/document-open.png</normaloff>:/res/document-open.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="DictPathLabel">
     <property name="text">
      <string>Katalog z plikami słownika:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="SamplePathLabel">
     <property name="text">
      <string>Katalog z próbkami głosowymi:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Ydpd.qrc"/>
 </resources>
 <connections/>
</ui>
